src/index.html

```diff
@@ -4,9 +4,349 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.css">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img class="header__logo" src="./images/logo.png"
+         alt="logo">
```

Comment:
Fix it everywhere where tag has more than 2 attributes
```suggestion
        <img 
          class="header__logo" 
          src="./images/logo.png"
          alt="logo"
       >
```